frontend/index.js

```diff
@@ -0,0 +1,118 @@
+                option.innerText = el; 
+                select.appendChild(option)
+            })
+        }
+    // in = index
+    // of = value
+    const getAllStudentsInClass = async (className) => {
+        try {
+
+            let studentsByClassDiv = document.querySelector("#studentsByClassDiv")
+            let res = await axios.get(`http://localhost:3000/class/${className}`);
+            let grabClass = res.data.allStudents;
+                grabClass.forEach((showStudent) => {
+                let {name, city, age, grade, house} = showStudent
+                const listStudent = `Class: ${className} Name: ${name} Age: ${age} City: ${city} Grade: ${grade} House: ${house}`
+                let p1 = document.createElement("p");
+                p1.innerText = listStudent;
+                studentsByClassDiv.appendChild(p1)
+    })
+} catch(error) {
+    console.log(error)
+    }
+}
+populateClasses()
+select.addEventListener("change", (event) => {
+   getAllStudentsInClass(event.target.value)
+  
+})
+    let newClassDisplay = document.querySelector("#newClassDisplay")
+    let newClassDisplayUL = document.querySelector("#newClassDisplayUL")
+    let addClassForm = document.querySelector("#addClassForm")
+    let enrollStudentForm = document.querySelector("#enrollStudentForm")
+    let newStudentDisplay = document.querySelector("#newStudentDisplay")
+    let newStudentDisplayUL = document.querySelector("#newStudentDisplayUL")
+    let failingForm = document.querySelector("#failingForm")
+
+
+
+addClassForm.addEventListener("submit", async (event) => {
+    event.preventDefault()
+    let className = document.querySelector("#className").value
+    let teacherName = document.querySelector("#teacherName").value
+try {
+    newClassDisplayUL.innerHTML = "";
+    let res = await axios.post(`http://localhost:3000/class/add`, {name: className, teacher: teacherName})
+    let h3 = document.createElement("h3");
+    h3.innerText = res.data.message
+    newClassDisplay.appendChild(h3)
+} catch (error) {
+    console.log(error)
+} 
+  })
+enrollStudentForm.addEventListener("submit", async (event) => {
+    event.preventDefault()
+    let sClassName = document.querySelector("#sClassName").value
+    let name = document.querySelector("#studentName").value
+    let age = document.querySelector("#studentAge").value
+    let city = document.querySelector("#studentCity").value
+    let grade = document.querySelector("#studentGrade").value
+    let house = document.querySelector("#studentHouse").value
+try {
+    newStudentDisplayUL.innerHTML = "";
+    let res = await axios.post(`http://localhost:3000/class/${sClassName}/enroll`,{class: sClassName, name: name, age: age, grade: grade, city: city, house: house})
+    let h3 = document.createElement("h3")
+    h3.innerText = res.data.message
+    newStudentDisplay.appendChild(h3)
+} catch (error){
+    console.log(error)
+}
+  })
+failingForm.addEventListener("submit", async (event) => {
```

Comment:
Preferably here the value of failing would have been a variable you can change by the value if its checked or not.